### DISCUSSION TITLE
ublobebm

### DISCUSSION THREAD

**Discussion by om3ga6400** [uBlockOrigin/uAssets#26509] [General] [answered]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.). Comment in the old issue threads even when they are closed or even if you have a different problem.
- [X] I did not remove any of the [default filter lists](https://github.com/user-attachments/assets/1879fc88-09bc-4de4-9e2f-3834a6ab396a), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
all
```

### Description

My friend is telling me about something called "ublobebm" dose this even exist and how would I get to it if it dose or is my friend just messing with me

### Other extensions used

na

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.61.2
Chromium: 131
filterset (summary):
 network: 271909
 cosmetic: 264730
 scriptlet: 57234
 html: 0
listset (total-discarded, last-updated):
 added:
  adguard-generic: 86003-7482, 3d.22h.33m
  adguard-mobile: 10013-99, 3d.22h.33m
  adguard-spyware-url: 1816-131, 3d.22h.33m
  adguard-spyware: 116530-686, 3d.22h.33m
  block-lan: 64-0, 6d.2h.49m
  curben-phishing: 495-0, 3h.20m
  adguard-cookies: 32840-3955, 3d.22h.33m
  ublock-cookies-adguard: 1980-0, 1h.9m
  fanboy-cookiemonster: 51775-144, 5d.2h.13m
  ublock-cookies-easylist: 1980-1980, 1h.9m
  adguard-social: 23260-1475, 3d.22h.32m
  fanboy-social: 16505-23, 1h.9m
  [11 lists not shown]: [too many]
 default:
  user-filters: 1-0, never
  ublock-filters: 40831-313, 3h.20m Δ
  ublock-badware: 12516-1, 3h.20m Δ
  ublock-privacy: 1666-23, 3h.20m Δ
  ublock-unbreak: 2544-0, 3h.20m Δ
  ublock-quick-fixes: 192-0, 3h.20m Δ
  easylist: 78629-979, 3h.20m Δ
  easyprivacy: 53316-31349, 3h.20m Δ
  urlhaus-1: 24960-0, 3h.20m
  plowe-0: 3540-0, 8d.22h.43m
filterset (user): [array of 1 redacted]
trustedset:
 removed:
  chrome-extension-scheme
  moz-extension-scheme
switchRuleset:
 added: [array of 2 redacted]
userSettings:
 advancedUserEnabled: true
 cloudStorageEnabled: true
 suspendUntilListsAreLoaded: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 2267 ms (selfie)
 maxAssetCacheWait: 379 ms
 cacheBackend: indexedDB
```
</details>

**Comment by MasterKia** [comment, marked as answer]
https://github.com/Blobby-Boi/uBlobeBM:

> uBlobeBM is a tool that can run bookmarklets using uBlock Origin. It's very useful if bookmarklets don't work on your device or if they are blocked by your administrator.

Interesting.

**Comment by om3ga6400** [comment]
> https://github.com/Blobby-Boi/uBlobeBM:
> 
> > uBlobeBM is a tool that can run bookmarklets using uBlock Origin. It's very useful if bookmarklets don't work on your device or if they are blocked by your administrator.
> 
> Interesting.

Thank you
